win_build/CUDA 11.8.props

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please keep current way CUDA is used. No need to create these additional files

Reply:
Removed the additional files.